src/postComment.jsx

```diff
@@ -0,0 +1,49 @@
+// child of comment card
+import axios from "axios"
+import { useState } from "react";
+
+
+export default function PostComment({article_id, commentList, setCommentList}) {
+    const [myusername, setmyUsername] = useState('')
+    const [mycomment, setmyComment] = useState('')
+    
+
+
+    function handleSubmit(event) {
+        event.preventDefault();
+         axios.post(`https://andrew-nc-news.onrender.com/api/articles/${article_id}/comments`, {username: myusername, body: mycomment}).then((response) => {
+          setCommentList([response.data.newComment, ...commentList]); 
+
+          setmyUsername("")
+          setmyComment("")
+         })
+      }
```

Comment:
The submit button still works while we wait for this post, meaning that you can press the submit button multiple times and make duplicate comments (which is especially a problem on a slow connection)

Ideally, we'd find a way around this!